pandas-stubs/core/indexes.pyi

```diff
@@ -16,14 +16,21 @@ class Index(Generic[_T]):
     @overload
     def __getitem__(self, idx: int) -> _T: ...
     @overload
+    def __getitem__(self, idx: Index[_T]) -> Index[_T]: ...
+    @overload
+    def __getitem__(self, idx: Series[bool]) -> _T: ...
```

Comment:
meh. you should use `.loc` instead for `Series[bool]`, but okay...

Reply:
```
error: "Index[str]" has no attribute "iloc"
error: "Index[str]" has no attribute "loc"
```

Reply:
This is the line for context
`list(data.columns.iloc[(data.apply(np.std) <= threshold)])`

Reply:
Oh I see. sorry